templates/zerver/for/open-source.md

```diff
@@ -1,104 +1,92 @@
+email address.
 [github-auth]: https://github.com/zulip/zulip/blob/7e9926233/zproject/prod_settings_template.py#L112
 
-* **Scale:** Zulip scales well to large communities — it's loved by
-  communities as large as 1000 users and beyond.
+**Quality data export**
+
+Our high quality export and import tools ensure you can always move from
+[zulipchat.com](https://zulipchat.com) hosting to your own servers.
+
+**Syntax highlighting**
+
+[Full Markdown support](/help/format-your-message-using-markdown), including
+syntax highlighting, makes it easy to discuss code, paste an error message,
+or make a complicated argument. Full LaTeX support as well.
+
+**Permalink from issues to chat**
+
+[Permalink to Zulip conversations](/help/share-a-message-or-conversation)
+from your issue tracker. Zulip’s topic-based threading keeps conversations
+coherent and organized.
+
+**Link from chat to issues**
+
+Efficiently refer to issues or code reviews with notation like `#1234` or
+`T1234`. You can set up any regex as a
+[custom linkification filter](/help/add-a-custom-linkification-filter) for
+your organization.
 
-## Zulip ❤​ open source
+**Hundreds of integrations**
 
-* **Free and open source software:** Zulip itself is open source,
-  [under the Apache License 2.0](https://github.com/zulip/zulip/blob/master/LICENSE).
+Get events from GitHub, Travis CI, JIRA, and
+[hundreds of other tools](/integrations) right in Zulip. Topics give each
+issue its own place for discussion.
 
-* **Free for open source:** Full, commercially-supported hosting at
-  zulipchat.com is free for open source communities.  [Sign up
-  today!](https://zulipchat.com/beta/))
+**Free and open source**
 
-<!--- This needs better — any! — documentation. -->
-* **Your data is yours:** Zulip's high-quality export and import tools
-  enable migrating a community smoothly from one host to another.
-  So if you start out running on zulipchat.com, you can always choose
-  later to move to your own servers, and vice versa.
+Don’t like something? You can open an issue, submit a patch, fork the
+project, or chat with us directly at
+[chat.zulip.org](https://chat.zulip.org). All code, including the desktop
+and mobile apps, is under the Apache 2 license.
 
-* **The open source leader:** Zulip is the leading open source group
-  chat option, with far more contributor momentum than any other.
-  Some numbers:
+**Proven model**
 
-    * Over 300 contributors in the Zulip server project alone — a
-      figure reached in only 18 months.
+Check out [chat.zulip.org](https://chat.zulip.org) to see Zulip in action
+for a project with ~5000 messages of developer discussion a week. The Zulip
+project is the largest and fastest growing open source group chat product,
+both by number of contributors (300+) and by commit velocity (more than
+Docker and Django combined).
 
-    * 75 people working on Zulip at the PyCon sprints in Portland in
-      May 2017.
+**Public archive (coming soon)**
```

Comment:
Ended up not doing this; we have an issue for it, but it isn't too inspiring. Agree it would be nice to link to something if/when we have it.